app/controllers/api/v1/payments_controller.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class PaymentsController < Api::BaseController
+      def create
+        result = ManualPayments::CreateService.call(
+          organization: current_organization,
+          params: create_params.to_h.deep_symbolize_keys
+        )
+
+        if result.success?
+          render(
+            json: ::V1::PaymentSerializer.new(result.payment, root_name: "payment")
+          )
+        else
+          render_error_response(result)
+        end
+      end
+
+      def index
+        result = PaymentsQuery.call(
+          organization: current_organization,
+          pagination: {
+            page: params[:page],
+            limit: params[:per_page] || PER_PAGE
+          },
+          filters: index_filters
+        )
+
+        if result.success?
+          render(
+            json: ::CollectionSerializer.new(
+              result.payments,
+              ::V1::PaymentSerializer,
+              collection_name: "payments",
+              meta: pagination_metadata(result.payments)
+            )
+          )
+        else
+          render_error_response(result)
+        end
+      end
+
+      def show
+        payment = Payment.for_organization(current_organization).find_by(id: params[:id])
```

Comment:
`.for_organzation` scope looks like a complex (and expensive) DB query to just find by id, I would replace it with...

```suggestion
        payment = current_organization.payments.find_by(id: params[:id])
```